scripts/check_backend_associated_test_file.py

```diff
@@ -71,6 +72,24 @@
 ]
 
 
+def _is_path_ignored(path_to_check: str) -> bool:
```

Comment:
We usually name functions as verbs (e.g. check_if_path_ignored)

Reply:
Fixed